WinUIGallery/Controls/SampleCodePresenter.xaml

```diff
@@ -10,31 +10,27 @@
     Loaded="SampleCodePresenter_Loaded"
     mc:Ignorable="d">
 
-    <Grid
-        Background="{x:Bind Background, Mode=OneWay}"
-        BorderBrush="{x:Bind BorderBrush, Mode=OneWay}"
-        BorderThickness="{x:Bind BorderThickness, Mode=OneWay}"
-        CornerRadius="{x:Bind CornerRadius, Mode=OneWay}">
-        <Grid.ColumnDefinitions>
-            <ColumnDefinition Width="*" />
-            <ColumnDefinition Width="Auto" />
-        </Grid.ColumnDefinitions>
+    <Grid Background="{x:Bind Background, Mode=OneWay}"
+          BorderBrush="{x:Bind BorderBrush, Mode=OneWay}"
+          BorderThickness="{x:Bind BorderThickness, Mode=OneWay}"
+          CornerRadius="{x:Bind CornerRadius, Mode=OneWay}">
         <ScrollViewer
             x:Name="CodeScrollViewer"
             VerticalAlignment="{x:Bind VerticalContentAlignment, Mode=OneWay}"
             HorizontalScrollBarVisibility="Auto"
             HorizontalScrollMode="Auto"
             VerticalScrollBarVisibility="Auto"
-            VerticalScrollMode="Auto">
+            VerticalScrollMode="Auto"
+            Loaded="CodeScrollViewer_Loaded">
             <ContentPresenter
                 x:Name="CodePresenter"
                 Margin="0,0,0,16"
+                Padding="16,0"
                 Loaded="CodePresenter_Loaded" />
         </ScrollViewer>
         <local:CopyButton
             x:Name="CopyCodeButton"
-            Grid.Column="1"
-            Margin="5,0,0,0"
+            Margin="0,0,8,0"
```

Comment:
This doesn't seem like a safe thing. A code example can be one line. We will have to place it at the very top.